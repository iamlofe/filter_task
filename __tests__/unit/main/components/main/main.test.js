import React from 'react';
import { shallow } from 'enzyme';

import Main from 'main/components/main/main';

describe('main component', () => {
    const baseProps = {
        history: {
            push: jest.fn()
        }
    };

    it('is shallow rendered without crashing', () => {
        const main = <Main {...baseProps} />;

        const maincomponent = shallow(main);
        expect(maincomponent).toMatchSnapshot();
    });
});
