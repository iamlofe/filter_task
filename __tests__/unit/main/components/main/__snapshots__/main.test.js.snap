// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main component is shallow rendered without crashing 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <div
    className="page-container"
  >
    <Switch>
      <Route
        component={[Function]}
        key="0"
        path="/"
      />
    </Switch>
  </div>
</Router>
`;
